### PR TITLE
Pipeline

### PR DESCRIPTION
**Pull request by Bojo992**
Adding basic pipeline that would run test an check if application can build

### CODE REVIEW

File: .github/workflows/basePipeline.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        with:
+          java-version: '23.0.0+37'
+          distribution: 'temurin'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4.0.0
+
+      - name: Check directory
+        run:
+          ls
```
Comment (Blake1412) [inline .github/workflows/basePipeline.yml]: should something be happening here?

File: .github/workflows/basePipeline.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+      - name: Setup JDK 23
+        uses: actions/setup-java@v4
+        with:
+          java-version: '23.0.0+37'
+          distribution: 'temurin'
+
+      - name: Check directory
+        run:
+          ls
```
Comment (Blake1412) [inline .github/workflows/basePipeline.yml]: Same as above

File: .github/workflows/basePipeline.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4.0.0
+
+      - name: Check directory
+        run:
+          ls
+
+      - name: Grant Permissions to gradlew
+        run:
+          cd demo;
```
Comment (Blake1412) [inline .github/workflows/basePipeline.yml]: Is 'demo' a placeholder name, shouldn't this be the name of the project folder? I believe it's 'project'

File: .github/workflows/basePipeline.yml (hunk 1 above)
Comment (Bojo992) [reply .github/workflows/basePipeline.yml]: it was adjustment

**Review by Blake1412 — APPROVED** [review]
lgtm

**Review by plutowootoe — APPROVED** [review]
looks good

**Comment by Bojo992** [comment]
[Identify Preliminary Database Schema](https://trello.com/c/tIhV0GqU/1-identify-preliminary-database-schema)

**Comment by Bojo992** [comment]
[Create Initial Pipelines](https://trello.com/c/3BE6yLnE/2-create-initial-pipelines)

**Pull request merged**